BitwardenShared/UI/Platform/DebugMenu/DebugMenuView.swift

```diff
@@ -0,0 +1,91 @@
+import SwiftUI
+
+// MARK: - DebugMenuView
+
+/// Represents the debug menu for configuring app settings and feature flags.
+///
+struct DebugMenuView: View {
+    // MARK: Properties
+
+    /// The store used to render the view.
+    @ObservedObject var store: Store<DebugMenuState, DebugMenuAction, DebugMenuEffect>
+
+    // MARK: View
+
+    var body: some View {
+        List {
+            Section {
+                featureFlags
+            } header: {
+                featureFlagSectionHeader
+            }
+        }
+        .toolbar {
+            ToolbarItem(placement: .navigationBarTrailing) {
+                Button {
+                    store.send(.dismissTapped)
+                } label: {
+                    Text(Localizations.close)
+                }
+                .accessibilityIdentifier("close-debug")
+            }
+        }
+        .navigationTitle("Debug Menu")
+        .task {
+            await store.perform(.viewAppeared)
+        }
+    }
+
+    /// The feature flags currently used in the app.
+    private var featureFlags: some View {
+        ForEach(store.state.featureFlags) { flag in
+            Toggle(
+                isOn: store.bindingAsync(
+                    get: { $0.isEnabled(for: flag.feature) },
+                    perform: { DebugMenuEffect.toggleFeatureFlag(flag.feature.rawValue, $0) }
```

Comment:
🎨 Could this be simplified by getting the flag's `isEnabled` value from the flag itself?
```suggestion
            Toggle(
                isOn: store.bindingAsync(
                    get: { _ in flag.isEnabled },
                    perform: { DebugMenuEffect.toggleFeatureFlag(flag.feature.rawValue, $0) }
```

Reply:
nice!  much cleaner.